e2e/test/scenarios/actions/actions-on-dashboards.cy.spec.js

```diff
@@ -961,6 +961,56 @@ const MODEL_NAME = "Test Action Model";
   );
 });
 
+describe("action error handling", { tags: ["@external", "@actions"] }, () => {
+  beforeEach(() => {
+    resetTestTable({ type: "postgres", table: TEST_TABLE });
+    restore("postgres-writable");
+    cy.signInAsAdmin();
+    resyncDatabase({ dbId: WRITABLE_DB_ID, tableName: TEST_TABLE });
+    createModelFromTableName({
+      tableName: TEST_TABLE,
+      modelName: MODEL_NAME,
+    });
+
+    cy.intercept("GET", "/api/action").as("getActions");
+    cy.intercept("GET", "/api/dashboard/*/dashcard/*/execute?parameters=*").as(
+      "executePrefetch",
+    );
+    cy.intercept("POST", "/api/dashboard/*/dashcard/*/execute").as(
+      "executeAPI",
+    );
```

Comment:
I find the names inconsistent with `e2e/test/scenarios/actions/actions-in-object-detail-view.cy.spec.js`. I do understand they're different since these are for dashcards.

However, we could compare these 2 aliases between 2 files
#### `e2e/test/scenarios/actions/actions-in-object-detail-view.cy.spec.js`
- `prefetchValues`
- `executeAction`

#### this file
- `executePrefetch`
- `executeAPI`

They are too different while they should be name almost the same in my opinion since they're doing the same thing, but on different pages that's why they're different APIs.

Reply:
baa4f8ea7e0f1d93853ac4e95ce50be48e946b93